install/kubernetes/cilium/templates/clustermesh-apiserver/deployment.yaml

```
@@ -204,11 +205,19 @@ spec:
               name: cilium-config
               key: enable-k8s-endpoint-slice
               optional: true
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: health
+          periodSeconds: 1
         {{- with .Values.clustermesh.apiserver.extraEnv }}
         {{- toYaml . | trim | nindent 8 }}
         {{- end }}
-        {{- if .Values.clustermesh.apiserver.metrics.enabled }}
         ports:
+        - name: health
```

```suggestion
        - name: apiserv-health
```

For consistency with the metrics port, and to prevent conflicts if also added to the other components in future.